app/javascript/dashboard/helper/filterQueryGenerator.js

```diff
@@ -8,11 +8,11 @@ function lowerCaseValues(operator, values) {
     });
   }
   return values;
-}
+};
 
 const generatePayload = data => {
   // Make a copy of data to avoid vue data reactivity issues
-  const filters = JSON.parse(JSON.stringify(data));
+  const filters = [...data];
```

Comment:
This won't work. It doesn't make a copy of the object. It just changes reference to the array

Reply:
Reverting, found out spread only does a shallow copy with root level keys